src/components/Tooltip/Tooltip.module.css

```diff
@@ -1,46 +1,17 @@
 .Tooltip {
-	position: relative;
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	max-width: 400px;
+	padding: 24px;
+	color: #ffffff;
+	background-color: #1d0c4a;
```

Comment:
Small nitpick - we have some CSS variables set up in `src/styles/main.css`. I noticed that this colour wasn't in there, but it looks like a reusable brand colour so it could be worth adding. Not necessary for approval, but just letting you know those variables are there and can be added to.